common4j/src/main/com/microsoft/identity/common/java/util/StringUtil.java

```diff
@@ -501,4 +501,15 @@ public static byte[] base64Decode(@NonNull final String input, int flags, @NonNu
             throw e;
         }
     }
+
+    /**
+     * Helper function to overwrite all characters in an array with zero
+     * @param chars
+     */
+    public static void overwriteWithZero(char[] chars) {
```

Comment:
nit: `final` on params here and everwhere

Reply:
Done